packages/backend-core/src/Base.ts

```diff
@@ -88,9 +94,10 @@ export class Base {
    * The public key will be supplied in the form of CryptoKey or will be loaded from the CLERK_JWT_KEY environment variable.
    *
    * @param {string} token
+   * @param {string[]} authorizedParties
    * @return {Promise<JWTPayload>} claims
    */
-  verifySessionToken = async (token: string): Promise<JWTPayload> => {
+  verifySessionToken = async (token: string, { authorizedParties }: VerifySessionTokenOptions = { authorizedParties: []}): Promise<JWTPayload> => {
```

Comment:
🙃 
For now we only have one option but can we create an object called `verifySessionTokenDefaultOptions` and use it here ?